src/form/examples/form-group.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.form.directive:pfFormGroup
+ * @restrict E
+ *
+ * @description
+ *  Encapsulates the structure and styling for a label + input used within a
+ *  Bootstrap3 based form.
+ *
+ *  This directive creates new scope.
+ *
+ * @param {string} pfLabel the text for the <label> element.
+ * @param {string} pfFieldId the id of the form field. Default value is id of the form field element.
+ * @param {string} pfLabelClass the class of the label element. Default value is "col-sm-2".
+ * @param {string} pfInputClass the class of the input element. Default value is "col-sm-5".
+ *
+ * @example
+ <example module="patternfly.form">
+
+   <file name="index.html">
+     <div ng-controller="FormDemoCtrl">
+       <p>Name: {{ item.name }}</p>
+       <p>Description: {{ item.description }}</p>
+       <form>
+         <pf-form-group pf-label="Name" required>
+           <input id="name" name="name" ng-model="item.name" type="text" required/>
+         </pf-form-group>
+         <pf-form-group pf-label="Description">
+           <textarea id="description" name="description" ng-model="item.description">
+             {{ item.description }}
+           </textarea>
+         </pf-form-group>
+       </form>
+       <p>Horizontal Form</p>
+       <form class="form-horizontal">
```

Comment:
Should this be updated to reflect the discussed changes to `pf-form-group` (i.e. to remove the horizontal form specific classes)?